wiki/Rules/Explicit_Content/id.md

```diff
@@ -0,0 +1,32 @@
+# Konten eksplisit
+
+![Tampilan beatmap yang mengandung konten eksplisit](img/explicit-tag.jpg "Contoh beatmap yang ditandai dengan label 'eksplisit'.")
+
+Ketika membuat beatmap, para mapper dapat menyematkan label 'eksplisit' pada beatmap-beatmap buatan mereka di situs web osu! untuk menandai bahwa beatmap-beatmap yang bersangkutan mengandung konten eksplisit.
+
+Artikel ini menjelaskan seputar berbagai hal yang terkait dengan dengan label tersebut beserta tata cara penggunaannya.
+
+## Hal-hal apa saja yang termasuk ke dalam konten eksplisit?
+
+osu! mengizinkan pemakaian **konten audio** yang bersifat eksplisit (baik dari segi *tema* ataupun *lirik* lagu) hingga batasan tertentu.
+
+**Di sisi lain, osu! tidak membenarkan penggunaan gambar dan video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
```

Comment:
```suggestion
**Di sisi lain, osu! tidak membenarkan penggunaan gambar maupun video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
```

Reply:
this because `maupun` could be related such `dan/atau`.

Reply:
kk applied